<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>452</width>
    <height>494</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>231</width>
     <height>23</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>账号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="userName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>密码：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="passwd"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextEdit" name="logInfo">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>12</y>
     <width>171</width>
     <height>112</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QGraphicsView" name="certImage">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>32</height>
       </size>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="certCode">
      <property name="maximumSize">
       <size>
        <width>150</width>
        <height>20</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QListWidget" name="coursesList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>411</width>
     <height>121</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="hintInfo">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>200</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>162</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5">
    <item>
     <widget class="QPushButton" name="getCourses">
      <property name="text">
       <string>查询课程</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="downloadAll">
      <property name="text">
       <string>下载并更新</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="refreshCertCode">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>100</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>刷新验证码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="choosePath">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>132</y>
     <width>120</width>
     <height>25</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>120</width>
     <height>25</height>
    </size>
   </property>
   <property name="text">
    <string>选择下载文件夹</string>
   </property>
  </widget>
  <widget class="QLabel" name="showPath">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>126</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QCheckBox" name="remPasswd">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>103</y>
     <width>116</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>记住密码</string>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>185</x>
     <y>100</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>登陆</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QTextBrowser" name="progressInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>380</y>
     <width>411</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
